client/view/Form/transformQuery.js

```diff
@@ -12,7 +12,7 @@ export function transformQuery(rawQuery) {
     } catch {}
   }
 
-  if (query.includes('\n')) {
+  if (query.includes('\n') && !query.includes('#')) {
```

Comment:
> tuned formatting. It will format multiline queries only if there are no comments

I am not sure that we need to tune formatting anymore.

We have it as a hack. Right now (on the example video) when you delete comment and website converts `,\n` to `,` (without a space even) it looks like a bug.

Let’s remove that old formatting tunning code.